frontend/src/components/structures/File.js

```diff
@@ -6,19 +6,21 @@ import {url} from 'core/config';
 function File(props) {
     // These functions should be moved out of this file.
     const renameFile = id => {
-        api.files.renameFile('user', id).then(response => {
```

Comment:
These API calls were broken

Reply:
Yes, because the backend was not finished at that point